invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -1,402 +1,712 @@
+          {"type": "number"},
+          {"type": "integer"},
+          {"type": "string"}
+        ]
+      }
+    },
+
+    "tombstone": {
+      "type": "object",
+      "description": "Tombstone for the record.",
+      "additionalProperties": false,
+      "properties": {
+        "reason": {
+          "type": "string",
+          "description": "Reason for removal."
+        },
+        "category": {
+          "type": "string",
+          "description": "Category for the removal."
+        },
+        "removed_by": {"$ref": "#/definitions/agent"},
+        "timestamp": {
+          "type": "string",
+          "description": "ISO8601 formatted timestamp in UTC.",
+          "format": "date-time"
         }
-      },
-      "extensions": {
-        "additionalProperties": {
-          "anyOf": [
-            {
-              "type": "array",
-              "items": {
-                "type": ["boolean", "number", "string"]
-              }
+      }
+    },
+
+    "provenance": {
+      "type": "object",
+      "description": "Record provenance.",
+      "additionalProperties": false,
+      "properties": {
+        "created_by": {"$ref": "#/definitions/agent"},
+        "on_behalf_of": {"$ref": "#/definitions/agent"},
+        "contact": {"$ref": "#/definitions/agent"}
+      }
+    },
+
+    "access": {
+      "type": "object",
+      "description": "Record access control and ownership.",
+      "additionalProperties": false,
+      "properties": {
+
+        "metadata": {
+          "description": "Metadata visibility (true - public, false - private)",
+          "type": "boolean"
+        },
+
+        "files": {
+          "description": "Files visibility (true - public, false - private)",
+          "type": "boolean"
+        },
+
+        "contact": {
+          "description": "Enable the contact feature to contact owners.",
+          "type": "boolean"
+        },
+
+        "owned_by": {
+          "description": "List of user IDs that are owners of the record.",
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true,
+          "items": {"$ref": "#/definitions/agent"}
+        },
+
+        "embargo_date": {
+          "description": "Embargo date of record (ISO8601 formatted date time in UTC). At this time both metadata and files will be made public.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "access_condition": {
+          "description": "Conditions under which access to files are granted",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "condition": {
+              "type": "string",
+              "description": "Textual description under which conditions access is granted."
             },
-            {"type": "boolean"},
-            {"type": "number"},
-            {"type": "string"}
-          ]
+            "default_link_validity": {
+              "type": "integer",
```

Comment:
If technically we go with a more fine-grained implementation (e.g. hours), maybe it's better to store this in seconds, to make it easy to uniformly turn into a `timedelta` object